lib/filter.g

```diff
@@ -344,4 +345,70 @@ BIND_GLOBAL( "CANONICAL_BASIS_FLAGS", QUO_INT(SUM_FLAGS,5) );
 
 #############################################################################
 ##
+#V  IdOfFilterByName
+#V  FilterByName
+##
+##  <#GAPDoc Label="FilterByName">
+##  <ManSection>
+##  <Func Name="IdOfFilterByName" Arg="name"/>
+##  <Func Name="FilterByName" Arg="name"/>
+##
+##  <Description>
+##  finds the id of the filter, or the filter with name <A>name</A> respectively,
+##  in the global FILTERS list. 
```

Comment:
I think it would be good to explain here why `FilterByName` is useful -- at first glance, `x := FilterByName("IsFinite");` is just a longer way of saying `x := IsFinite`. But of course it's possible to have filters which are not bound to a global variable with matching names. So perhaps state just that: That a filter has an intrinsic name, and there need not be a global variable with matching name; and even if there is such a gvar, it need not be bound to that filter.

Also perhaps have a "see also" ref pointing to `CategoryByName`?